package.json

```diff
@@ -62,63 +62,64 @@
   "dependencies": {
     "frontend-collective-react-dnd-scrollzone": "^1.0.1",
     "lodash.isequal": "^4.5.0",
-    "prop-types": "^15.6.1",
-    "react-dnd": "^6.0.0 || ^7.0.0",
-    "react-dnd-html5-backend": "^7.0.1",
-    "react-lifecycles-compat": "^3.0.4",
+    "prop-types": "^15.7.0",
+    "react-dnd": "^7.1.0",
+    "react-dnd-html5-backend": "^7.1.0",
+    "react-lifecycles-compat": "^3.0.0",
     "react-sortable-tree": "^2.6.0",
-    "react-virtualized": "^9.19.1"
+    "react-virtualized": "^9.21.0"
   },
   "peerDependencies": {
     "react": "^16.3.0",
-    "react-dnd": "^6.0.0 || ^7.0.0",
+    "react-dnd": "^7.1.0",
     "react-dom": "^16.3.0"
   },
   "devDependencies": {
-    "@babel/cli": "^7.1.5",
-    "@babel/core": "^7.1.6",
-    "@babel/plugin-transform-modules-commonjs": "^7.1.0",
-    "@babel/preset-env": "^7.1.6",
+    "@babel/cli": "^7.2.0",
+    "@babel/core": "^7.3.0",
+    "@babel/plugin-transform-modules-commonjs": "^7.2.0",
+    "@babel/preset-env": "^7.3.0",
     "@babel/preset-react": "^7.0.0",
-    "@storybook/addon-options": "^4.0.0-alpha.4",
-    "@storybook/addon-storyshots": "^4.0.0-alpha.4",
-    "@storybook/react": "^4.0.0-alpha.4",
-    "autoprefixer": "^9.3.1",
+    "@storybook/addon-options": "^5.0.0",
+    "@storybook/addon-storyshots": "^5.0.0",
+    "@storybook/react": "^5.0.0",
+    "autoprefixer": "^9.4.10",
     "babel-core": "^7.0.0-bridge.0",
     "babel-eslint": "^10.0.1",
-    "babel-jest": "^23.6.0",
-    "babel-loader": "^8.0.4",
-    "codesandbox": "^1.2.10",
+    "babel-jest": "^24.1.0",
+    "babel-loader": "^8.0.5",
+    "codesandbox": "^2.1.0",
+    "core-js": "^2.6.5",
```

Comment:
I added socks and core-js because it resolved to some historical versions with issues and npm does not have resolutions field for these purposes.